R/woods_plot.R

```diff
@@ -0,0 +1,82 @@
+#' Wood's plot
+#'
+#' Creates a Wood's plot that plots log2 fold change of peptides or precursors along the protein sequence.
+#'
+#' @param data Data frame containing differential abundance, start and end peptide or precursor positions, protein 
+#' length and optionally a variable based on which peptides or precursors should be colored. 
+#' @param fold_change Column in the data data frame containing log2 fold changes.
+#' @param start_position Column in the data frame containing the start positions for each peptide or precursor.
+#' @param end_position Column in the data frame containing the end positions for each peptide or precursor.
+#' @param protein_length Column in the data frame containing the length of the protein.
+#' @param protein_id Optional argument, column in the data frame containing protein identifiers. Required if only one protein 
+#' should be plotted and the data frame contains only information for this protein.
+#' @param facet Optional argument, column in the data frame containing information by which data should be faceted. This can be 
+#' protein identifiers. 
+#' @param coloring Optional argument, column in the data frame containing information by which peptide or precursors should
```

Comment:
Do you want to use BE or AE? If we stick to BE you should call it colouring and use "ou" everywhere you mention color.